templates/ehub-existing-setup.json

```diff
@@ -0,0 +1,267 @@
+{
```

Comment:
Can we have a single template that uses existing ehub, but creates one if the given one doesn't exist?